### ISSUE TITLE
Curve Frame node

### ISSUE THREAD

**Issue by Durman** [nortikin/sverchok#3307]
![2020-06-02_15-06-15](https://user-images.githubusercontent.com/28003269/83513298-a1a91b80-a4e2-11ea-9f17-979e80998043.png)

<details> 
  <summary>Error</summary>  

```Text
Traceback (most recent call last):
  File "D:\╨б╨╡╤А╨╡╨│╨╣\╨п╨╜╨┤╨╡╨║╤Б╨Ф╨╕╤Б╨║\3.╨б╨▓╨╛╨╕_╨┐╤А╨╛╨╡╨║╤В╤Л\Blender_code\addons\sverchok\core\update_system.py", line 383, in do_update_general
    node.process()
  File "D:\╨б╨╡╤А╨╡╨│╨╣\╨п╨╜╨┤╨╡╨║╤Б╨Ф╨╕╤Б╨║\3.╨б╨▓╨╛╨╕_╨┐╤А╨╛╨╡╨║╤В╤Л\Blender_code\addons\sverchok\nodes\curve\curve_frame.py", line 55, in process
    matrices_np, normals, binormals = curve.frame_array(ts)
  File "D:\╨б╨╡╤А╨╡╨│╨╣\╨п╨╜╨┤╨╡╨║╤Б╨Ф╨╕╤Б╨║\3.╨б╨▓╨╛╨╕_╨┐╤А╨╛╨╡╨║╤В╤Л\Blender_code\addons\sverchok\utils\curve.py", line 186, in frame_array
    raise e
  File "D:\╨б╨╡╤А╨╡╨│╨╣\╨п╨╜╨┤╨╡╨║╤Б╨Ф╨╕╤Б╨║\3.╨б╨▓╨╛╨╕_╨┐╤А╨╛╨╡╨║╤В╤Л\Blender_code\addons\sverchok\utils\curve.py", line 179, in frame_array
    matrices_np = np.linalg.inv(matrices_np)
  File "<__array_function__ internals>", line 6, in inv
  File "C:\Users\User\Desktop\blender-2.83-af3a55bb1b7d-windows64\2.83\python\lib\site-packages\numpy\linalg\linalg.py", line 551, in inv
    ainv = _umath_linalg.inv(a, signature=signature, extobj=extobj)
  File "C:\Users\User\Desktop\blender-2.83-af3a55bb1b7d-windows64\2.83\python\lib\site-packages\numpy\linalg\linalg.py", line 97, in _raise_linalgerror_singular
    raise LinAlgError("Singular matrix")
numpy.linalg.LinAlgError: Singular matrix
```

</details>

[test2.zip](https://github.com/nortikin/sverchok/files/4716708/test2.zip)

**Comment by portnov** [comment]
Actually this is the same problem as in #3306.

**Comment by portnov** [comment]
I think this can be considered as fixed by #3336 .
